pkg/controllers/apps/sidekick_controller.go

```diff
@@ -69,6 +71,8 @@ type SidekickReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.0/pkg/reconcile
 func (r *SidekickReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	fmt.Println("reconlice called with -------------->", req.NamespacedName)
```

Comment:
Remove all the unnecessary logs.
`fmt.Println()`, `klog.Info()`, `klog.Error()`. There are many.